### DISCUSSION TITLE
404 .wasm file not found but exists in GitHub Pages Repo

### DISCUSSION THREAD

**Discussion by leoplaw** [community/community#30903] [Other Feature Feedback, Questions, & Ideas] [answered]
I've pushed my ``dist`` folder to the ``gh-pages`` branch on on my project repo and configured pages to serve / deploy from the root of this branch.

[https://tradex-app.github.io/TradeX-chart/](https://tradex-app.github.io/TradeX-chart/)

However on page load it cannot find a .wasm file that it requires.

``XHR GET https://tradex-app.github.io/TradeX-chart/node_modules/talib-web/lib/talib.wasm [HTTP/2 404 Not Found 404ms]``

Checking the repo branch, I see that the "missing" file does exist.

[https://github.com/tradex-app/TradeX-chart/tree/gh-pages/node_modules/talib-web/lib](https://github.com/tradex-app/TradeX-chart/tree/gh-pages/node_modules/talib-web/lib)

I've tried clearing browser caches on multiple browsers, but still to no avail, and still resulting in 404 not found.

I've tried various other suggestions / tricks I found via searches, but still no joy.

Can anyone here tell me how to fix this?

**Comment by jfullstackdev** [comment]
the correct link is this, I don't know how you arrived with that folder structure, but here it is

<https://tradex-app.github.io/TradeX-chart/assets/node_modules/talib-web/lib/talib.wasm>

if you click that, the file will be downloaded, on my end, it's Chrome, so I doubt if you want to embed that because it will always download

**Comment by leoplaw** [comment, marked as answer]
Seems my ``vite.config`` was incorrectly configured resulting in a bad build.
